### PR TITLE
Update syntax.mdx

### PR DESCRIPTION
**Pull request by parkchoongho**
fix wrong grammar

### CODE REVIEW

File: website/docs/language/resources/syntax.mdx [hunk 1]
```diff
@@ -133,7 +133,7 @@ The following meta-arguments are documented on separate pages:
 
 ## Custom Condition Checks
 
-You can use `precondition` and `postcondition` blocks to specify assumptions and guarantees about how the resource operates. The following examples creates a precondition that checks whether the AMI is properly configured.
+You can use `precondition` and `postcondition` blocks to specify assumptions and guarantees about how the resource operates. The following examples create a precondition that checks whether the AMI is properly configured.
```
Comment (laurapacilio) [inline website/docs/language/resources/syntax.mdx:136]:
```suggestion
You can use `precondition` and `postcondition` blocks to specify assumptions and guarantees about how the resource operates. The following example creates a precondition that checks whether the AMI is properly configured.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Thank you for catching this! I have a tiny suggestion as I think we want to refer to the entire thing as a singular example.

**Comment by parkchoongho** [comment]
> Thank you for catching this! I have a tiny suggestion as I think we want to refer to the entire thing as a singular example.

Oh, I think that's a better one!

**Review by laurapacilio — APPROVED** [review]
🚀

**Pull request merged**